### ISSUE TITLE
How to train unregular box?

### ISSUE THREAD

**Issue by Joyako** [IDEA-Research/detrex#294]
Hi, my data set labels are unregular, where to modify the data preprocessing code?

**Comment by rentainhe** [comment]
You can try to register your own data following detectron2's official documentation here: https://detectron2.readthedocs.io/en/latest/tutorials/datasets.html

**Comment by Joyako** [comment]
@rentainhe thank you for your reply, does data augmentation need to be modified?

**Comment by rentainhe** [comment]
> @rentainhe thank you for your reply, does data augmentation need to be modified?

Yes, we use detectron2's default data augmentation, but it only works on the normal box, you should double-check this in detectron2's source code.

**Comment by Joyako** [comment]
> > @rentainhe thank you for your reply, does data augmentation need to be modified?
> 
> Yes, we use detectron2's default data augmentation, but it only works on the normal box, you should double-check this in detectron2's source code.

thanks, I got it

**Comment by rentainhe** [comment]
> > > @rentainhe thank you for your reply, does data augmentation need to be modified?
> > 
> > 
> > Yes, we use detectron2's default data augmentation, but it only works on the normal box, you should double-check this in detectron2's source code.
> 
> thanks, I got it

You're welcome, detectron2's documentation has detailed tutorial on how to register your own dataset and augmentations, it may be helpful
